### PR TITLE
Added University of Padova and minor updates to two other professors

### PR DESCRIPTION
**Pull request by lambertoballan**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

**Comment by emeryberger** [comment]
Please resolve conflicts.

**Comment by lambertoballan** [comment]
It should be ok now.

File: csrankings.csv [hunk 1]
```diff
@@ -403,6 +404,7 @@ Alessandro Moschitti,Qatar Computing Research Institute,http://www.qcri.org/our-
 Alessandro Orso,Georgia Institute of Technology,http://www.cc.gatech.edu/home/orso,wCfYkMkAAAAJ
 Alessandro Redondi,Politecnico di Milano,http://home.deib.polimi.it/redondi,8ka5NmUAAAAJ
 Alessandro Salandrino,University of Kansas,http://www.eecs.ku.edu/people/faculty,ZX0I5pcAAAAJ
+Alessandro Sperduti,University of Padova,https://www.math.unipd.it/~sperduti,JsExaWMAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:407]: Department of Math is not Computer Science.

Comment (emeryberger) [reply csrankings.csv:407]: Unless this person has a cross-appointment in CS and can thus solely advise a PhD student _for a CS PhD_, they are not eligible for inclusion.

File: csrankings.csv [hunk 2]
```diff
@@ -827,9 +829,11 @@ Andrea Lockerd,Georgia Institute of Technology,http://www.cc.gatech.edu/~athomaz
 Andrea Lockerd Thomaz,Georgia Institute of Technology,http://www.cc.gatech.edu/~athomaz,jIs-Y2gAAAAJ
 Andrea Maggiolo Schettini,University of Pisa,http://www.di.unipi.it/~maggiolo,NOSCHOLARPAGE
 Andrea Marino,University of Pisa,http://pages.di.unipi.it/marino,FUKaLzkAAAAJ
+Andrea Pietracaprina,University of Padova,http://www.dei.unipd.it/~capri,X4HU5HIAAAAJ
 Andrea S. LaPaugh,Princeton University,https://www.cs.princeton.edu/~aslp,NOSCHOLARPAGE
 Andrea Tagliasacchi,University of Victoria,http://gfx.uvic.ca,1RmD-YsAAAAJ
 Andrea Thomaz,Georgia Institute of Technology,http://www.cc.gatech.edu/~athomaz,jIs-Y2gAAAAJ
+Andrea Vedaldi,University of Oxford,http://www.robots.ox.ac.uk/~vedaldi,bRT7t28AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:836]: Robotics is not CS. See above comment. This person and others below are not listed as faculty in CS (http://www.cs.ox.ac.uk/people/faculty.html).

File: csrankings.csv [hunk 3]
```diff
@@ -1446,7 +1451,6 @@ B. G. Kim,University of Massachusetts Lowell,http://www.cs.uml.edu/~kim,ujXxdMIA
 B. John Oommen,Carleton University,http://www.scs.carleton.ca/~oommen,E5TiDR4AAAAJ
 B. Prabhakaran,University of Texas at Dallas,http://www.utdallas.edu/~praba,BENEK78AAAAJ
 B. R. Badrinath,Rutgers University,http://www.cs.rutgers.edu/~badri,ZuCDNqwAAAAJ
-B. S. Manjunath,University of California - Santa Barbara,https://vision.ece.ucsb.edu/people/bs-manjunath,wRYM4qgAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:1449]: Why the removal?

File: csrankings.csv [hunk 4]
```diff
@@ -1749,7 +1754,6 @@ Bing Qin 0001,Harbin Institute of Technology,http://ir.hit.edu.cn/~qinb,LKnCub0A
 Bing Wang 0001,University of Connecticut,http://www.engr.uconn.edu/~bing,HvNdFJcAAAAJ
 Bing Xie,Peking University,http://sei.pku.edu.cn/~xiebing,f66ae1YAAAAJ
 Bing Xue,Victoria University of Wellington,https://ecs.victoria.ac.nz/Main/BingXue,htnpub8AAAAJ
-Bing Zeng,UESTC,http://faculty.uestc.edu.cn/cengbing/en/index.htm,s-kUGYQAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:1752]: Spurious removal.

File: csrankings.csv [hunk 5]
```diff
@@ -2675,7 +2680,6 @@ Claudia Eckert,TU Munich,https://www.sec.in.tum.de/claudia-eckert,wGd3KhEAAAAJ
 Claudia Hauff,TU Delft,http://www.wis.ewi.tudelft.nl/hauff,gMUKVkkAAAAJ
 Claudia Linnhoff-Popien,LMU Munich,http://www.mobile.ifi.lmu.de/team/claudia-linnhoff-popien,bJBAUu8AAAAJ
 Claudia Neuhauser,University of Minnesota,http://cbs.umn.edu/contacts/claudia-neuhauser,_3qZty0AAAAJ
-Claudia Plant,University of Vienna,https://cs.univie.ac.at/claudia.plant,QSYzTd0AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:2678]: Spurious removal.

File: csrankings.csv [hunk 6]
```diff
@@ -2688,6 +2692,7 @@ Claudio Gutierrez,Universidad de Chile,https://www.dcc.uchile.cl/cgutierr,jzeqUi
 Claudio Gutiérrez 0001,Universidad de Chile,https://www.dcc.uchile.cl/cgutierr,jzeqUi4AAAAJ
 Claudio Luis de Amorim,UFRJ,http://www.cos.ufrj.br/index.php/pt-BR/pessoas/details/18/1007-amorim,YQNnYZQAAAAJ
 Claudio Orlandi,Aarhus University,http://www.cs.au.dk/~orlandi,EVDZd5oAAAAJ
+Claudio Palazzi,University of Padova,https://www.math.unipd.it/~cpalazzi,PBpX1HAAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:2695]: See above.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Lots of issues. I stopped at 3538. Please undo all spurious removals and remove all ineligible faculty.

File: csrankings.csv (hunk 1 above)
Comment (lambertoballan) [reply csrankings.csv:407]: This is a Math & CS Department (in Italy this is not the only example). All the faculties from Univ. of Padova that are included here can advise PhD students and they are teaching in the bachelor, master and PhD program in computer science.

File: csrankings.csv (hunk 3 above)
Comment (lambertoballan) [reply csrankings.csv:1449]: You are right, this is noi intentional.

File: csrankings.csv (hunk 4 above)
Comment (lambertoballan) [reply csrankings.csv:1752]: You are right, this is not intentional.

File: csrankings.csv (hunk 5 above)
Comment (lambertoballan) [reply csrankings.csv:2678]: You are right, this is not intentional.

File: csrankings.csv (hunk 6 above)
Comment (lambertoballan) [reply csrankings.csv:2695]: See above, same comment about Math & CS at U Padova. This is a reference: https://www.math.unipd.it/en/courses/phd/

File: csrankings.csv (hunk 2 above)
Comment (lambertoballan) [reply csrankings.csv:836]: Added A. Vedaldi and A. Zisserman both from robotics institute at Oxford which has a huge intersection with CS (a similar example is the robotic dept. at CMU). They advise plenty of PhD students from CS and they are two of the most important people in computer vision and machine learning worldwide.

Comment (emeryberger) [reply csrankings.csv:836]: Unless they are able to _solely_ advise students _in CS_, they do not meet the inclusion criteria. That is the case for members of the Robotics dept at CMU.

File: csrankings.csv (hunk 6 above)
Comment (emeryberger) [reply csrankings.csv:2695]: I see two departments here.
https://www.math.unipd.it/en/courses/phd/
https://www.dei.unipd.it/home-page
Can you clarify?

Comment (lambertoballan) [reply csrankings.csv:2695]: The Math (&CS) department is the major department for CS at the Univ. of Padova. However, there is also a subset of people from the DEI dept (Information Engineering) that are affiliated to the school of Engineering but are doing research, supervising PhD students and teaching in Informatics.

Comment (emeryberger) [reply csrankings.csv:2695]: Unless they can solely advise PhD students in the CS department, they are not eligible for inclusion.

**Comment by emeryberger** [comment]
No response in two weeks. Closing - feel free to reopen.

**Pull request closed without merging**